packages/components/ContextualMenu/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@fluentui-react-native/contextual-menu",
+  "version": "0.1.0",
+  "description": "A cross-platform ContextualMenu component using the Fluent Design System",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "just": "fluentui-scripts",
+    "clean": "fluentui-scripts clean",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "verify-api": "fluentui-scripts verify-api-extractor",
+    "update-api": "fluentui-scripts update-api-extractor"
+  },
+  "dependencies": {
+    "@uifabricshared/foundation-compose": "^1.6.51",
+    "@fluentui-react-native/adapters": ">=0.3.13 <1.0.0",
+    "@fluentui-react-native/interactive-hooks": ">=0.4.20 <1.0.0",
+    "@fluentui-react-native/text": ">=0.5.49 <1.0.0",
+    "@fluentui-react-native/tokens": ">=0.4.48 <1.0.0"
+  },
+  "devDependencies": {
+    "@office-iss/react-native-win32": "0.62.0-preview.3",
```

Comment:
If you are able to get rid of the `IViewWin32*` usage (I think that's the only usage), don't forget to get rid of it here :)